x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -67,300 +130,509 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
-                    {
-                      defaultMessage: 'normal',
-                    }
-                  ),
-                  'data-test-subj': 'mlModelsStartDeploymentModalNormalPriority',
-                },
-              ]}
-              data-test-subj={'mlModelsStartDeploymentModalPriority'}
-            />
-          </EuiFormRow>
-        </EuiDescribedFormGroup>
-      ) : null}
+      }
+    } else if (
+      (cloudInfo.isCloud && !cloudInfo.isMlAutoscalingEnabled) ||
+      (!cloudInfo.isCloud && showNodeInfo)
+    ) {
+      // Running in cloud with autoscaling disabled or on-prem
+      if (config.adaptiveResources) {
+        // With adaptive resources
+        switch (config.vCPUUsage) {
+          case 'low':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.lowCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to one vCPU, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+              }
+            );
+          case 'medium':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.mediumCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to two vCPUs, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+              }
+            );
+          case 'high':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.highCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level may use the maximum number of vCPUs available for this deployment from the Cloud console. If the maximum is 2 vCPUs or fewer, this level is equivalent to the medium or low level.',
+              }
+            );
+        }
+      } else {
+        // Without adaptive resources
+        switch (config.vCPUUsage) {
+          case 'low':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.lowCpuStaticeHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to the minimum required for ELSER to run if supported by your Cloud console selection. It may not be sufficient for a production application.',
+              }
+            );
+          case 'medium':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.mediumCpuStaticeHelp',
+              {
+                defaultMessage:
+                  'Your model will consume {maxVCPUs} vCPUs consistently, if provided by the Cloud console.',
```

Comment:
No need to mention cloud console, vCPU limits are know in advance in this case. 
```suggestion
                  'Your model will consume {maxVCPUs} vCPUs consistently.',
```